core/modules/views/templates/views.theme.inc

```diff
@@ -930,10 +930,14 @@ function template_preprocess_views_view_list(&$variables) {
   $handler  = $variables['view']->style_plugin;
 
   $class = explode(' ', $handler->options['class']);
-  $class = array_map('backdrop_clean_css_identifier', $class, array(array()));
+  foreach ($class as $key => $individual) {
+    $class[$key] = backdrop_clean_css_identifier($individual, array());
+  }
 
   $wrapper_class = explode(' ', $handler->options['wrapper_class']);
-  $wrapper_class = array_map('backdrop_clean_css_identifier', $wrapper_class, array(array()));
+  foreach ($wrapper_class as $key => $individual) {
```

Comment:
Same as my previous comment; can we rename `$wrapper_class` to `$wrapper_classes` instead, and do:
```suggestion
  foreach ($wrapper_classes as $key => $wrapper_class) {
```